cardano-node/src/Cardano/Tracing/Config.hs

```diff
@@ -179,7 +179,7 @@ traceConfigParser v ctor =
       chainSyncProtocol :: OnOff TraceChainSyncProtocol
       chainSyncProtocol = OnOff False
       connectionManager :: OnOff TraceConnectionManager
-      connectionManager = OnOff False
+      connectionManager = OnOff True
```

Comment:
So I think the correct place to add defaults would be in `defaultPartialNodeConfiguration` due to: https://github.com/input-output-hk/cardano-node/blob/e5d32b3558f436d3178296d19b23578524681839/cardano-node/src/Cardano/Node/Run.hs#L115

Because then the user can override the tracers if they want to via the config yaml file.

I don't think you can actually turn off the tracing if you change the values here.

Reply:
oh, there are some other tracers with the same problem, you might want to look into it at some point.

Reply:
I'd like to leave it out as is; there seems to be work to be done to make it right.  One cannot use `Last` for entire tracing config (`pncTracingConfig :: Last TraceOptions`), since when one is passing config it will almost always overwrite all values.  Instead, `pncTracingConfig` should have type `TraceOptions` and each of its field should be wrapped in `Last`, and a monoid instance for `TraceOptions` should be derived generically.  Doing this is beyond this pr.

Reply:
Yes this is beyond the scope of the PR.

Reply:
I created an issue for this https://github.com/input-output-hk/cardano-node/issues/3534